File: .vscode/Lógica e Pensamento Computacional/Desafios/bin/Desafio.cs
//Para ler e escrever dados em Java, aqui na DIO padronizamos da seguinte forma:
// * new Scanner(System.in): cria um leitor de Entradas, com métodos úteis com prefixo "next";
// * System.out.println: Imprime um texto de Saída (Output) e pulando uma linha.

using System;

public class Desafio
{

    public static void main()
    {
        //lê os valores de Entrada
        float valorSalario = float.Parse(Console.ReadLine());
        float valorBeneficios = float.Parse(Console.ReadLine());

        float valorImposto = 0;
        if (valorSalario >= 0 && valorSalario <= 1100) 
        {
            //Atribui a aliquota de 5% mediante o salário:
            valorImposto = 0.05F * valorSalario;
        }
        else if (valorSalario >= 1100.01 && valorSalario <= 2500.00) {
            //Atribui a aliquota de 10% mediante o salário:
            valorImposto = 0.10F * valorSalario;
        } 
        else 
        {
            //Atribui a aliquota de 15% mediante o salário:
            valorImposto = 0.15F * valorSalario;
        }
        //TODO Criar as demais condições para as aliquotas em 10.00% e 15.00%.

        //Calcule e imprima a Saída (com 2 casas decimais):
        float saída = valorSalario - valorImposto * valorBeneficios;
        Console.WriteLine(saida.ToString("0.00"));
    }
}

File: .vscode/Lógica e Pensamento Computacional/Desafios/src/Desafio.kt
object ReceitaFederal {
    fun calcularImposto(salario: Double): Double {
        val aliquota = when {
            (salario >= 0 && salario <= 1100) -> 0.05
            (salario >= 1100.01 && salario <= 2500.00) -> 0.10
            else -> 0.15 TODO("Criar as condições para as aliquotas de 10.00% e 15.00%.")
        }
        return aliquota * salario
    
    }
}

fun main () {
    val valorSalario = readLine()!!.toDouble();
    val valorBeneficios = readLine()!!.toDouble();

    val valorImposto = ReceitaFederal.calcularImposto(valorSalario);
    val saida = valorSalario - valorImposto + valorBeneficios;

    println(String.format("%.2F", saida));

}